### PR TITLE
Request ownership change for Dynatrace plugin

### PR DESCRIPTION
**Pull request by johannes-b**
## Hey, I just made a Pull Request!

Request ownership change from the Dynatrace plugin as discussed and agreed with the original creator, TELUS.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .github/CODEOWNERS
Comment (vinzscam) [inline .github/CODEOWNERS:41]: probably we would need to create a new `@backstage/dynatrace-reviewers` team, as I don't think it's possible to specify an entire organization as code owner 🤔

**Comment by benjdlambert** [comment]
Hey @johannes-b! Thanks for wanting to take ownership of this! Just looking at some of the options that we have available to us, how would Dynatrace feel about following what we're going to do with the PagerDuty plugin https://github.com/backstage/backstage/issues/19347 and moving it out into your own Org in Github instead and publishing under your own npm namespace? :pray:

**Comment by johannes-b** [comment]
> Hey @johannes-b! Thanks for wanting to take ownership of this! Just looking at some of the options that we have available to us, how would Dynatrace feel about following what we're going to do with the PagerDuty plugin #19347 and moving it out into your own Org in Github instead and publishing under your own npm namespace? 🙏

Hi @benjdlambert! For the time being, we would like to keep the plugin where it is but we should definitely address this topic again in some months from now.

**Comment by johannes-b** [comment]
Hello @benjdlambert, is there something I can do to get this PR approved?

**Comment by benjdlambert** [comment]
Hey @johannes-b! So after chatting with the other maintainers, our governance has some obstacles involved here with the `CODEOWNERS` change. We're thinking that it would make sense to re-label everything in here as Dynatrace like the rest of the PR does, but maybe we can hold off on the `CODEOWNERS` file change until either the code is moved out to a repository that you own, or we get some folks together to go through steps to become [Org Members](https://github.com/backstage/backstage/blob/master/GOVERNANCE.md#organization-member) by first off managing the code in the repo for a period of time.

I'm sure you'll understand that we don't want to break the Governance rules that are setup here to protect plugins that are contributed to this repo, so there's a few hurdles that we have to jump through :)

**Comment by johannes-b** [comment]
> Hey @johannes-b! So after chatting with the other maintainers, our governance has some obstacles involved here with the `CODEOWNERS` change. We're thinking that it would make sense to re-label everything in here as Dynatrace like the rest of the PR does, but maybe we can hold off on the `CODEOWNERS` file change until either the code is moved out to a repository that you own, or we get some folks together to go through steps to become [Org Members](https://github.com/backstage/backstage/blob/master/GOVERNANCE.md#organization-member) by first off managing the code in the repo for a period of time.
> 
> I'm sure you'll understand that we don't want to break the Governance rules that are setup here to protect plugins that are contributed to this repo, so there's a few hurdles that we have to jump through :)

I totally understand and don`t want to push against the Governance rules. However, I faced a failed GH action when I opened the PR since the entry in the `CODEOWNERS` file was missing. Please see here: https://github.com/backstage/backstage/actions/runs/6338908930/job/17217001201

If I revert my changes in the CODEOWNERS file, I will stuck in the the same issue. Is there something I can change?

**Comment by benjdlambert** [comment]
Ah I guess it want's `spec.owner` to be reflected in the `CODEOWNERS` file. It's a little unfortunate, but I think you'll have to revert that to `maintainers` for now.

**Review by benjdlambert — APPROVED** [review]
Nice thanks :tada:

**Pull request merged**
